docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md

```diff
@@ -0,0 +1,211 @@
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L127-L131
+```
+
+Perform the relevant checks, such as confirm the number of batches.  For example (be sure to adjust this code to match
+the specifics of your data and configuration):
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L137-L138
+```
+
+Note that in the introspection mode, all included tables will contribute rows to the composition of the batch data
+representing the splits.
+
+## Partitioning
+
+In the partitioning mode, a dedicated data asset is assigned to a table, and `Splitting` is used to associate the
+resulting batches with desired semantics.  Partitioning is fully specified in the `tables` section of the configuration.
+
+For the present example, we can split according to the "passenger_count" column with the focus on two-passenger rides:
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L142-L146
+```
+
+Perform the relevant checks, such as confirm the number of batches.  For example (be sure to adjust this code to match
+the specifics of your data and configuration):
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/files/yaml_example.py#L152-L153
+```
+
+## Sampling
+
+`Sampling` provides a means for reducing the amount of data in the retrieved batch to facilitate data analysis.
+
+To configure `Sampling`, specify a dimension (i.e., `column_name` or the entire `table`), the method of sampling, and
+parameters to be used by the specified sampling method.
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L159-L163
+```
+
+We can then obtain a random `10%` of the rows in the batch:
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L169
+```
+
+Finally, confirm the expected number of batches was retrieved and the reduced size of a batch (due to sampling):
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L170-L176
+```
+
+## Additional Notes
+
+Available `Splitting` methods and their configuration parameters:
+
+    +-----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	+
```

Comment:
@alexsherstinsky I'm not sure why this isn't getting rendered into docusaurus format (I see it as a code block in the PR rendered output) but you might want to double check the syntax: https://www.markdownguide.org/extended-syntax/#tables - it also shows an example of the rendered table.

Reply:
@anthonyburdi Thank you for pointing this out.  I will be revisiting this document once the files one is merged (as the latter informs some of the document organization structure of this document as well).  Thanks!